XCast/XCast.h

```diff
@@ -101,6 +101,9 @@ class XCast : public AbstractPlugin, public RtNotifier {
     static bool m_enableStatus;
     //Timer related variables and functions
     TpTimer m_locateCastTimer;
+
+    const std::string systemRequestKeyValue = "vaLib)6?b%$^CP;/F";
```

Comment:
I would move it to the anonymous namespace in the implementaton file. It is not necessary to pollute the header file with this name.